### PR TITLE
add a runnable example and document how verbosity/silence works

### PR DESCRIPTION
**Pull request by grosser**
fixes https://github.com/capistrano/sshkit/issues/456 since `verbosity: :DEBUG` does what I need
also fixed a few headings since ordering was level-2 -> level-4

@will-in-wi

### CODE REVIEW

**Comment by grosser** [comment]
hmmm the "Finished in" line still appears I'll try to remove that too ...

**Comment by grosser** [comment]
ah nvm ... but upload needs a tweak

**Comment by grosser** [comment]
ok, upload uses the passed `verbosity:` now too ... looks good ?

File: examples/simple_connection.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# tiny example so you can play with the sshkit or make a failing example for an issue
+require 'bundler/setup'
```
Comment (mattbrictson) [inline examples/simple_connection.rb]: Why is this needed?

Comment (grosser) [reply examples/simple_connection.rb]: it's meant as a starting point to debug sshkit itself, so it needs to use the local checkout and not the installed gem

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ appear at the top.
 ## [Unreleased][]
 
   * Your contribution here!
+  * [#468](https://github.com/capistrano/sshkit/pull/468): Make `upload!` take a `:verbosity` option like `exec` does - [@grosser](https://github.com/grosser)
```
Comment (mattbrictson) [inline CHANGELOG.md]: 👍

File: README.md [hunk 3]
```diff
@@ -115,6 +115,11 @@ the raised error.
 Helpers such as `runner()` and `rake()` which expand to `execute(:rails, "runner", ...)` and
 `execute(:rake, ...)` are convenience helpers for Ruby, and Rails based apps.
 
+### Verbosity / Silence
+
+ - raise verbosity of a command: `execute "echo DEAD", verbosity: :ERROR`
+ - hide a command from output: `execute "echo HIDDEN", verbosity: :DEBUG`
```
Comment (mattbrictson) [inline README.md]: Thanks for documenting this!
